modules/prebidServerBidAdapter/index.js

```diff
@@ -571,10 +587,29 @@ const OPEN_RTB_PROTOCOL = {
             bidRequest.serverResponseTimeMs = serverResponseTimeMs;
           }
 
+          const extPrebidTargeting = utils.deepAccess(bid, 'ext.prebid.targeting');
+
+          // If ext.prebid.targeting exists, add it as a property value named 'adserverTargeting'
+          if (extPrebidTargeting && typeof extPrebidTargeting === 'object') {
+            bidObject.adserverTargeting = extPrebidTargeting;
```

Comment:
what's the use case for PBS setting the keys into Prebid.js? AFAIK this is only for mobile use case where the client doesn't process the bids => keys.

Reply:
There are additional KVPs that need to be copied through to the ad server:
- hb_cache_host
- hb_cache_path
- hb_winurl

Reply:
I see thanks for clarifying. I wonder if PBJS should be setting these then? Because there could be a case where PBS isn't involved but universal creative is being used.

Reply:
I realized after commenting while the above is true, you most likely won't need either `_host` or `_path` unless you are using AMP or prebid mobile, both of which are powered by PBS. So this most likely safe.